src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -996,18 +1051,17 @@ def test_fw_codeblock_1(fort_writer):
 def test_fw_codeblock_2(fort_writer):
     '''Check the FortranWriter class codeblock method correctly prints out
     the Fortran representation when there is a code block that is part
-    of a line (not a whole line). In this case the ":" in the array
-    access is a code block.
+    of a line (not a whole line). In this case the data initialisation
+    of the scalar "(/ 0.0 /)" is a code block.
 
     '''
     # Generate fparser2 parse tree from Fortran code.
     code = (
         "module test\n"
         "contains\n"
-        "subroutine tmp(a,n)\n"
-        "  integer, intent(in) :: n\n"
-        "  real, intent(out) :: a(n,n,n)\n"
-        "    a(2,n,:) = 0.0\n"
+        "subroutine tmp()\n"
+        "  real a\n"
```

Comment:
Strictly speaking, shouldn't `a` be declared as an array?

Reply:
Yes. Updated.